### PR TITLE
Update Programming.md

### PR DESCRIPTION
**Pull request by KrishGaur1354**
Signed-off-by: ThatOneKrish <csrealracing13d@gmail.com>

### CODE REVIEW

**Comment by KrishGaur1354** [comment]
Could you label PR under `hacktoberfest-accepted`

File: Programming.md [hunk 1]
```diff
@@ -52,6 +52,7 @@
 ## JS
 
 1. [Discord JS](https://github.com/discordjs/discord.js)
+2. [Intro to JS](https://www.geeksforgeeks.org/introduction-to-javascript/)
```
Comment (closebracket) [inline Programming.md:55]: Good.

File: Programming.md [hunk 2]
```diff
@@ -60,6 +61,7 @@
 2. [Tailwind CSS Docs](https://tailwindcss.com/docs/)
 3. [Aweomesheet](https://zombiefox.github.io/awesomeSheet/) - CSS Generator
 4. [Learn CSS](https://web.dev/learn/css/) - Free Google course on CSS
+5. [CSS Units](https://www.geeksforgeeks.org/css-units/)
```
Comment (closebracket) [inline Programming.md:64]: Good.

File: Programming.md [hunk 3]
```diff
@@ -68,6 +70,15 @@
 ### HTML
 
 1. [W3 HTML Introduction](https://www.w3schools.com/html/)
+2. [GFG HTML Basics](https://www.geeksforgeeks.org/html-basics/)
```
Comment (closebracket) [inline Programming.md:73]: Good.

File: Programming.md [hunk 4]
```diff
@@ -68,6 +70,15 @@
...
+3. [MDN HTML Docs](https://developer.mozilla.org/en-US/docs/Web/HTML)
```
Comment (closebracket) [inline Programming.md:74]: Good.

File: Programming.md [hunk 5]
```diff
@@ -68,6 +70,15 @@
...
+
+### Java
+1. [w3 schools Java Info](https://www.w3schools.com/java/default.asp)
```
Comment (closebracket) [inline Programming.md:77]: Good.

File: Programming.md [hunk 6]
```diff
@@ -68,6 +70,15 @@
...
+2. [w3 schools Java Examples(Beginners & Intermediates)](https://www.w3schools.com/java/java_examples.asp)
```
Comment (closebracket) [inline Programming.md:78]: This is pretty much the same as the one above. Please provide a different source of examples.

File: Programming.md [hunk 7]
```diff
@@ -68,6 +70,15 @@
...
+3. [Java-Projects-For-Beginners](https://github.com/KrishGaur1354/Java-Projects-for-Beginners)
```
Comment (closebracket) [inline Programming.md:79]: This wouldn't really qualify as a resource of good enough quality to be added yet. If you could develop the resource, you are welcome to add it at a later date.

File: Programming.md [hunk 8]
```diff
@@ -68,6 +70,15 @@
...
+4. [GeeksForGeeks Java documentation](https://www.geeksforgeeks.org/java/)
```
Comment (closebracket) [inline Programming.md:80]: Good.

File: Programming.md [hunk 9]
```diff
@@ -68,6 +70,15 @@
...
+5. [Java Bootcamp by Kunal Kushwaha on Github](https://github.com/kunal-kushwaha/DSA-Bootcamp-Java)
```
Comment (closebracket) [inline Programming.md:81]: Compared to the other resource you tried to add, this is a much better example. I will approve this one.

**Review by closebracket — CHANGES_REQUESTED** [review]
Please see comments in the review. Thank you.

**Pull request closed without merging**